client/jira.go

```diff
@@ -308,6 +348,11 @@ func newCreateIssueRequest(projectKey, issueType, summary string) IssueRequest {
 	return IssueRequest{Fields: fields}
 }
 
+func prepare(transitionId string) DoTransitionRequest {
```

Comment:
Same comment, I need this shape of request body:
```
{ 
	"transition": {
		"id": "21"
	}
}

```

Reply:
I think what @pamelin is saying here is that it the argument and parameter does not need to be `transitionId` and can be simplified to `id` since it's just the binding of the variable in the program's memory.

`func prepare(id string) ... `

Similarily:
```go
type TransitionIdRequest struct {
		Id string `json:"id"`
}
```
We can just call the field `Id`.  For maintaing the shape only the nesting level and the json encoding is important. Hope this makes sense 🤔

Reply:
done